docker-serve

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash 
+
```

Comment:
Hey @valmayaki , could you please shed some light on what this line does

Reply:
This is what is called a shebang. When you execute the script is make the shell to know which environment required to execute the instructions in the script e.g for a ruby script you would see "#!/usr/local/bin/ruby -w" or for php you would see "#!/usr/bin/env php", so that when you run it (when you call ./docker-serve in the shell) you do not need to precede it with "bash ./docker-serve" or if its a php script you wouldn't need to run "php script-with-shebang". Especially if an extension is not provided.